docs/components/MessagingInputView.jsx

```diff
@@ -141,6 +157,12 @@ export default class MessagingInputView extends React.PureComponent {
             description: "If true, manually disables the send button",
             optional: true,
           },
+          {
+            name: "replyTo",
+            type: "React.ReactNode",
+            description: "Optional prop to use for message reply content",
+            optional: true,
+          },
```

Comment:
Think you had written this part before you added the `onReplyCancel` prop

Reply:
oh shoot good catch